.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: scala
+scala:
+  - 2.11.7
+jdk:
+  - oraclejdk7
```

Comment:
I'd say we should, otherwise we're running against whatever the travis default jdk is (in this case it happens to be Oracle JDK 7).